tests/integration/Vcs/GitTest.php

```diff
@@ -95,6 +95,8 @@ public function testThatCommitsSinceTheLastVersionAreReturned(): void
     public function testThatCommitsSinceTheFirstCommitAreReturnedIfNoReleasesExist(): void
     {
         $git = new Git();
+
+        $this->expectException(RepositoryNotFoundException::class);
```

Comment:
This exception should not be thrown here if I look at the test naming.

Reply:
Without this line the test doesn't pass, I don't know what's going wrong?

Reply:
This has been reverted